src/agent/agent.c

```diff
@@ -1815,6 +1817,21 @@ static bool ensure_orch_address(Agent *agent) {
                 return false;
         }
 
+        if (isIPv6Addr(agent->host)) {
+                hirte_log_errorf("IPV6 address '%s' is not support at moment", agent->host);
```

Comment:
hm, iiuc `get_address` does support ipv6, no? So is there a reason we don't support ipv6?